tools/manifestfile.py

```diff
@@ -492,8 +528,12 @@ def main():
         except ManifestFileError as er:
             print(er, file=sys.stderr)
             exit(1)
+    print(m.metadata())
```

Comment:
Is this intentional, to always print out the metadata?  What do existing users of this script expect to be printed out when it's run?

Reply:
Yes. This is entirely for debugging manifest files.

Reply:
OK